src/plugins/intel_npu/src/backend/src/zero_device.cpp

```diff
@@ -84,6 +85,10 @@ ZeroDevice::ZeroDevice(const std::shared_ptr<ZeroInitStructsHolder>& initStructs
 
     // Find the corresponding command queue group.
     _group_ordinal = zeroUtils::findGroupOrdinal(command_group_properties, device_properties);
+
+    zeroUtils::throwOnFail(
+        "pfnDeviceGetGraphProperties2",
+        _graph_ddi_table_ext->pfnDeviceGetGraphProperties2(_initStructs->getDevice(), &graph_properties));
```

Comment:
I've done a quick check on Ubuntu using [this driver](https://af01p-ir.devtools.intel.com/artifactory/drivers_vpu_linux_client-ir-local/engineering-drops/driver/main/release/24ww7.1.1/) (ww07, not supporting graph extension v1.6) and the run end in segmentation fault.

Reply:
One more note: Even after fixing this issue, I believe some extra validation using an older driver version (e.g. one relying on graph extension v1.5) would be helpful.